src/config.ts

```diff
@@ -1,37 +1,37 @@
-import dotenv from "dotenv";
-dotenv.config();
+import dotenv from 'dotenv'
+dotenv.config()
 
 interface DbConfig {
-  username: string;
-  password: string;
-  database: string;
+  username: string
+  password: string
+  database: string
 }
 
 interface Config {
-  db: DbConfig;
-  apiKey: String;
-  apiUrl: String;
-  internalApiKey: String;
+  db: DbConfig
+  apiKey: string
+  apiUrl: string
+  internalApiKey: string
 }
 
-function readEnvVariable(key: string): string {
-  const value = process.env[key];
-  if (!value) {
-    throw new Error(`Missing required environment variable: ${key}`);
+function readEnvVariable (key: string): string {
+  const value = process.env[key]
+  if (value == null) {
```

Comment:
Always prefer type checking as well, in this case we should use `===` instead of `==`. Additionally, no need to check specifically for null value, you can check it using `!value`.

Reply:
This was changed by eslint when strictBooleanExpression was allowed (!value was throwing that error connected to strict Boolean expressions), I will check everywhere and change it.